include/alpaka/mem/buf/uniformCudaHip/Set.hpp

```diff
@@ -1,4 +1,5 @@
-/* Copyright 2019 Benjamin Worpitz, Erik Zenker, Matthias Werner, René Widera
+/* Copyright 2021 Benjamin Worpitz, Erik Zenker, Matthias Werner, René Widera,
+ *                Andrea Bocci
```

Comment:
Why the line break here? Did clang-format do this?

Reply:
I copied the line break length from `include/alpaka/mem/buf/uniformCudaHip/Copy.hpp`.
I an check if `clang-format` is happy with having all names on the same line and use that.
Which one do you prefer ?

Reply:
I would keep it on the same line.

Reply:
`clang-format` is happy to have all names on one line.

Do you prefer 
```suggestion
/* Copyright 2021 Benjamin Worpitz, Erik Zenker, Matthias Werner, René Widera, Andrea Bocci
```
?

Reply:
I've put all names back on one line.